main.h

```diff
@@ -0,0 +1,40 @@
+namespace TelCoColorCoder {
```

Comment:
main file is used to call test code. But main.h contains production code.
This filename maybe renamed to reflect the content/intention of the code. 
For ex ColorCodeConvertor - which indicates something about code conversion.